src/main/java/reiff/brickbreaker/ManyNetworks.java

```diff
@@ -0,0 +1,139 @@
+
+    public ManyNetworks(Ball ball, Paddle paddle, Controller controller) {
+        this.ball = ball;
+        this.paddle = paddle;
+        this.controller = controller;
+    }
+
+    //Here we are creating 1000 networks and storing them in an array.
+    public NeuralNetwork [] generateNetworks() {
+
+        NeuralNetwork[] neuralNetworksArray = new NeuralNetwork[1000];
+
+        for (int i = 0; i < 1000; i++) {
+            neuralNetworksArray[i] = new NeuralNetwork(2, 2, 4, 2);
+        }
+
+        return neuralNetworksArray;
+    }
+    //now we have an array of 1000 network, one in each position.
+
+    public NeuralNetwork[] createNextGeneration(NeuralNetwork[] topPerformingNetworks) {
+        NeuralNetwork[] nextGeneration = new NeuralNetwork[1000];
+
+        int index = 0;
+
+        for (int i = 0; i < topPerformingNetworks.length; i++) {
+
+            for (int j = 0; j < topPerformingNetworks.length; j++) {
+                if (i != j) {  // Don't merge the network with itself
+                    NeuralNetwork parent1 = topPerformingNetworks[i];
+                    NeuralNetwork parent2 = topPerformingNetworks[j];
+
+                    NeuralNetwork child = parent1.merge(parent2);
+                    child.mutate(0.1);
+
+                    nextGeneration[index] = child;
+                    index++;
+
+
+                    if (index >= nextGeneration.length) {
+                        return nextGeneration;
+                    }
+                }
+            }
+        }
+
+        return nextGeneration;
+    }
+
+    //We need to figure out now, where are we letting these new networks play and so on...
+
+    //Here we are going to let each network play
+    public Map<NeuralNetwork, Double> networksPlay(NeuralNetwork[] neuralNetworksArray) {
+
+        Map<NeuralNetwork, Double> performanceMap = new HashMap<>();
+
+        for (NeuralNetwork neuralNetwork : neuralNetworksArray) {
+
+            controller.startGame();
+
+            long startTime = 0;
+            while (!controller.isGameStopped()) {
+
+                startTime = System.nanoTime();
+
+                controller.updateBallPosition();
+
+                double[] input = new double[1];
+                input[0] = calculateAngle(ball.getX(), ball.getY(), paddle.getX(), paddle.getWidth());
+                double[] answer = neuralNetwork.guess(input);
+
+                // answer[0] corresponds to probability of going left and answer[1] of going right
+                double movementSpeed = Math.abs(answer[0] - answer[1]);
+                if (answer[0] > answer[1]) {
+                    movePaddleLeft(movementSpeed); //Ilana implement
+                } else {
+                    movePaddleRight(movementSpeed); //Ilana implement
+                }
+
+            }
+
+            long endTime = System.currentTimeMillis();
+            double timeSurvived = (endTime - startTime) / 1000.0;
+
+            if (controller.won()) {
```

Comment:
You didn't care at this point if the NN has won, you care how long it lived for.